bitcoin-death-cross/README.md

```diff
@@ -0,0 +1,3 @@
+# bitcoin-death-cross
+
+This demo only works on a single member
```

Comment:
Can we enhance this README to include build and run instructions. Also high level overview would be also helpful

Reply:
There will be a blog.hazelcast.com entry, and once published I'll copy it to README.md